src/callbacks_stage/positivity_adaptive_filter.jl

```diff
@@ -0,0 +1,60 @@
+# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
+# Since these FMAs can increase the performance of many numerical algorithms,
+# we need to opt-in explicitly.
+# See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
+@muladd begin
+
+struct FilterParam{FactorType<:Real}
```

Comment:
The FilterParam here is not really necessary in this PR (1D DGMulti). I imagine for shock capturing we will have a different set of relaxation factors or parameters, and we can then define `ShockCaptureParam <: Param` and passed that along like
```
AdaptiveFilter{Param}
  param::Param
end
```
What do you think?